### DISCUSSION TITLE
Rate limit error

### DISCUSSION THREAD

**Discussion by gilbertlucas46** [ErickLimaS/anime-website#108] [Q&A]
in your consumet api are you using proxy? do you think cors-anywhere will work with it?

PORT=Port number of the server. (optional)
BILIBILI_COOKIE=Cookie for Bilibili (optional)
PROXY=Proxies for requests (optional) (e.g. ["https://proxy1.com", "https://proxy2.com"])
REDIS_HOST=Redis host (optional)
REDIS_PORT=Redis port (optional)
REDIS_PASSWORD=Redis password (optional)
NINE_ANIME_HELPER_URL=9anime (optional)
TMDB_KEY=TMDB access token auth (optional)
RECAPTCHATOKEN=Gogoanime recaptcha token (optional)
GOGOANIME_URL=Gogoanime custom url (optional)
ZORO_URL=Zoro custom url (optional)
NODE_ENV=Environment (optional) (DEMO or PROD)

**Comment by ErickLimaS** [comment]
No. I just did the deploy with default values.
